library/nxos_switchport

```diff
@@ -116,16 +130,25 @@ options:
 '''
 EXAMPLES = '''
 # Config a switchport to be a trunk interface with native vlan 10 and carriers vlan 2-100
-- nxos_switchport: interface=Ethernet1/1 mode=trunk native_vlan=10 trunk_vlans=2-100 host={{ inventory_hostname }}
+- nxos_switchport: interface=Ethernet1/1 mode=trunk native_vlan=10 trunk_vlans=2-100 host={{ inventory_hostname }} state=present
+
+# Config a switchport to be a trunk interface and add vlan 101-120 as allowed
+- nxos_switchport: interface=Ethernet1/1 mode=trunk trunk_vlans=101-120 host={{ inventory_hostname }} state=present
 
 # Config a switchport to an access port on vlan 20
-- nxos_switchport: interface=Ethernet1/2 mode=access access_vlan=20 host={{ inventory_hostname }}
+- nxos_switchport: interface=Ethernet1/2 mode=access access_vlan=20 host={{ inventory_hostname }} state=present
```

Comment:
FYI, state=present should still be default, so no need to have it here, but I have no issues keeping it.